redash/models.py

```diff
@@ -1000,6 +1006,32 @@ def outdated_queries(cls):
 
         return outdated_queries.values()
 
+    @classmethod
+    def delete_stale_resultsets(cls):
+        delete_count = 0
+        queries = Query.query.filter(Query.schedule_resultset_size != None).order_by(Query.schedule_resultset_size.desc())
+        # Multiple queries with the same text may request multiple result sets
+        # be kept. We start with the one that keeps the most, and delete both
```

Comment:
Do we `start with the one that keeps the most` because we want to limit how much deleting is done in `cleanup_query_results()`? If so, maybe a comment on that? I was confused at first why we only look at the first query but the deleting limit listed in `cleanup_query_results()` is my best guess at why.

Reply:
We start with the one that keeps the most because if we start with any others it'd delete results that need to be kept.